samples/bluetooth/subrating/README.rst

```diff
@@ -0,0 +1,225 @@
+    I: Identity: FD:AA:B8:5A:36:E2 (random)
+    I: HCI: version 5.4 (0x0d) revision 0x1224, manufacturer 0x0059
+    I: LMP: version 5.4 (0x0d) subver 0x1224
+    Bluetooth initialized
+    Choose device role - type c (central) or p (peripheral):
+    Peripheral. Starting advertising
+    Advertising successfully started
+    Connected as peripheral
+    Service discovery completed
+    Subrate changed. Subrate Factor: 1, Continuation number: 0
+    Simulating burst of data.
+    Response round-trip time: 12542 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16876 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16876 us
+    Response round-trip time: 16845 us
+    Subrate changed. Subrate Factor: 10, Continuation number: 0
+    Simulating burst of data.
+    Response round-trip time: 182434 us
+    Response round-trip time: 196746 us
+    Response round-trip time: 196777 us
+    Response round-trip time: 196746 us
+    Response round-trip time: 196777 us
+    Response round-trip time: 196746 us
+    Response round-trip time: 196746 us
+    Response round-trip time: 296752 us
+    Response round-trip time: 196777 us
+    Response round-trip time: 196746 us
+    Subrate changed. Subrate Factor: 10, Continuation number: 1
+    Simulating burst of data.
+    Response round-trip time: 12451 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16815 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16876 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16845 us
+    Response round-trip time: 16876 us
+    Response round-trip time: 16845 us
+
+
+Dependencies
+*************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`latency_readme`
+* :ref:`latency_client_readme`
+
+This sample uses the following `sdk-nrfxlib`_ libraries:
+
+* :ref:`nrfxlib:softdevice_controller`
+
+In addition, it uses the following Zephyr libraries:
+
+* :file:`include/console.h`
+* :ref:`zephyr:kernel_api`:
+
+  * :file:`include/kernel.h`
+
+* :file:`include/sys/printk.h`
+* :file:`include/zephyr/types.h`
+* :ref:`zephyr:bluetooth_api`:
+
+  * :file:`include/bluetooth/bluetooth.h`
+  * :file:`include/bluetooth/conn.h`
+  * :file:`include/bluetooth/gatt.h`
+  * :file:`include/bluetooth/hci.h`
+  * :file:`include/bluetooth/uuid.h`
+  * :file:`include/bluetooth/scan.h`
+  * :file:`include/bluetooth/gatt_dm.h`
+
+
+References
+***********
+
+For more information about LE Connection Subrating, see the following chapters of the `Bluetooth Core Specification`_:
+
+* Vol 6, Part B, 5.1.19 Connection Subrate Update procedure
+* Vol 6, Part B, 5.1.20 Connection Subrate Request procedure
```

Comment:
Might be good to reference `4.5.1 Connection events` as this has some nice subrating diagrams